README.md

```diff
@@ -166,6 +167,10 @@ Download & install:
 
 Social images (social media share image, Twitter header and generic banner) can be generated by running `yarn build:images` while your dev server is running. They will be placed in the static folder, in the language you defined in your config.json (make sure `LOCALE`, `BASE_URL` and `COUNTRY_CODE` is set correctly). The social-image.png is then automatically linked as social media share image.
 
+### Generation localized word-list
```

Comment:
And this 🙈

Reply:
Sry, Ondra is php programmer... Also is very good translater.

Reply:
👍 no problem. The script was nice, and if we had had to generate the lists it would have been useful